ui/lib/pki/addon/templates/issuers/index.hbs

```diff
@@ -0,0 +1,76 @@
+<Toolbar>
+  <ToolbarActions>
+    <ToolbarLink @params={{array "configuration.create.import-ca"}}>
+      Import
+    </ToolbarLink>
+    <BasicDropdown @class="popup-menu" @horizontalPosition="auto-right" @verticalPosition="below" as |D|>
+      <D.Trigger class={{concat "toolbar-link" (if D.isOpen " is-active")}} @htmlTag="button">
+        Generate
+        <Chevron @direction="down" @isButton={{true}} />
+      </D.Trigger>
+      <D.Content @defaultClass="popup-menu-content">
+        <nav class="box menu">
+          <ul class="menu-list">
+            <li class="action">
+              <LinkTo @route="configuration.create.generate-root" {{on "click" D.actions.close}}>
```

Comment:
Heads up that using the on click modifier with the dropdown close action seems to cause issues in production builds as identified in #16983. I'm not sure if these links would be impacted but it would be a good idea to test in a production build or update this to trigger the close action in the next tick of the runloop (see #16983 for example).

Reply:
Thank you! Modified.